gengo/src/analysis/mod.rs

```diff
@@ -42,21 +65,8 @@ impl Analysis {
     }
 }
 
-pub struct Iter<'map>(IndexMapIter<'map, PathBuf, Entry>);
-
-impl<'map> Iterator for Iter<'map> {
-    type Item = (&'map PathBuf, &'map Entry);
-
-    fn next(&mut self) -> Option<Self::Item> {
-        self.0.next()
-    }
-}
-
-impl<'map> IntoIterator for &'map Analysis {
-    type Item = (&'map PathBuf, &'map Entry);
-    type IntoIter = Iter<'map>;
-
-    fn into_iter(self) -> Self::IntoIter {
-        self.iter()
+impl std::fmt::Debug for Analysis {
```

Comment:
Nitpick: Why prefix with `std::fmt::`? `Debug` is one of the default imports AFAIK.
```suggestion
impl Debug for Analysis {
```

Reply:
There is no reason in particular. It seems you are working on this branch right now (just pushed an update for windows), and I welcome that you make changes as you see fit.